ruby-lsp.gemspec

```diff
@@ -12,9 +12,7 @@ Gem::Specification.new do |s|
   s.homepage = "https://github.com/Shopify/ruby-lsp"
   s.license = "MIT"
 
-  s.files = Dir.chdir(File.expand_path(__dir__)) do
-    %x(git ls-files -z).split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|misc)/}) }
-  end
+  s.files = Dir.glob("{lib,exe}/**/*") + ["README.md", "CHANGELOG.md", "Gemfile", "VERSION", "LICENSE.txt"]
```

Comment:
I don't think we should bundle the `Gemfile`, it is not used in deployed gems at all and will be just dead code. I see now that I've done the same mistake of bundling it in Tapioca, which I will fix.